google-home-community.groovy

```diff
@@ -2443,7 +2443,7 @@ private deviceStateForTrait_Dock(deviceTrait, device) {
     ]
 }
 
-@SuppressWarnings(['UnusedPrivateMethod', 'MethodSize'])
+@SuppressWarnings(['UnusedPrivateMethod', 'LineLength'])
```

Comment:
Please break your long lines up instead of suppressing this warning.

Reply:
Fixed